job/server/src/test/java/alluxio/job/plan/replicate/SetReplicaDefinitionEvictTest.java

```diff
@@ -85,8 +89,9 @@ private Set<Pair<WorkerInfo, SetReplicaTask>> selectExecutorsTestHelper(
     BlockInfo blockInfo = new BlockInfo().setBlockId(TEST_BLOCK_ID);
     blockInfo.setLocations(blockLocations);
     when(mMockBlockStore.getInfo(TEST_BLOCK_ID)).thenReturn(blockInfo);
-    PowerMockito.mockStatic(BlockStoreClient.class);
-    PowerMockito.when(BlockStoreClient.create(mMockFileSystemContext)).thenReturn(mMockBlockStore);
+    mMockStaticBlockStore = mockStatic(BlockStoreClient.class);
+    mMockStaticBlockStore.when(() -> BlockStoreClient.create(mMockFileSystemContext))
+        .thenReturn(mMockBlockStore);
```

Comment:
Is it possible to hoist this initialization into the `before` method? Like in `SetReplicaDefinitionReplicateTest`.

Reply:
done